templates/nsq/0/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+services:
+  nsqlookupd01:
+    image: nsqio/nsq:v1.0.0-compat
+    command: /nsqlookupd
```

Comment:
Affinity rule (soft or hard) to assure distribution across the host.
```
+    labels:
+      io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/nsqlookupd02,io.rancher.stack_service.name=$${stack_name}/nsqlookupd03
```

Reply:
good point LGTM.